### PR TITLE
Create new topic Prestashop

### PR DESCRIPTION
**Pull request by alelazcano**
initial info - Create new topic Prestashop

### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> PrestaShop is currently used by 300,000 shops worldwide and is available in 60 different languages.

### CODE REVIEW

**Review by Nguyenpham89 — COMMENTED** [review]
> #3632

File: topics/prestashop/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+aliases: prestashop,presta shop
+created_by: Bruno Lévêque, Igor Schlumberger
+display_name: PrestaShop
+github_url: https://github.com/prestashop
+logo: prestashop.png
+related: platform
+released: 2007
+short_description: PrestaShop is a open source e-commerce platform. PrestaShop is currently used by 300,000 shops worldwide and is available in 60 different languages.
+topic: php,mysql,symfony,ecommerce,smarty,prestashop
```
[the remark this answers is not in the document]
Comment (Nguyenpham89) [reply topics/prestashop/index.md:10]: IEFT

[the remark this answers is not in the document]
Comment (Nguyenpham89) [reply topics/prestashop/index.md:10]: > 'topic' key should match the directory name 'prestashop'.

**Comment by Str77781** [comment]
Hello,

I have a question for you. I'm looking to put together a team. A small
business.

No joking. What if I was able to show that I am Satoshi Nakamoto in 15
minutes. I need a small team of 10 people or leass, to help explain the
best ways to get my wallets in the best source for everyone. Would that be
METAMASK? BITCOIN WALLET ??? THINGS LIKE THAT.

WOULD YOU BE INTERESTED IN TALKING FURTHER? YOU CAN CALL ME OR TEXT. TEXT
IS A HOOD WAY TO LET ME KNOW YOU'RE NOT A TELEMARKETER..
THANKS FOR YOUR TIME

PLEASE CALL ME.
SCOT RINGA
3193203284

File: topics/prestashop/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+aliases: prestashop,presta shop
```
Comment (Kristories) [inline topics/prestashop/index.md:2]: Alias 'prestashop' must not be the same as the topic.

File: topics/prestashop/prestashop.png
Comment (Kristories) [inline topics/prestashop/prestashop.png:1]: The actual image is 512px wide. It should be 288px.

File: topics/prestashop/index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+aliases: prestashop16,prestashop17,prestashop8,prestashop-ecommerce,project-prestashop
+created_by: Bruno Lévêque, Igor Schlumberger
+display_name: PrestaShop
+github_url: https://github.com/prestashop
+logo: prestashop.png
+related: platform
+released: 2007
+short_description: PrestaShop is a open source e-commerce platform. PrestaShop is currently used by 300,000 shops worldwide and is available in 60 different languages.
```
Comment (Kristories) [inline topics/prestashop/index.md:9]: `short_description` no more than 130 characters

**Comment by alelazcano** [comment]
@Kristories I was just working on it. The truth is that years ago I did not have to do PR, my role as project manager has made me forget some things. I appreciate his predisposition and accompaniment, really, he is very helpful to me 💯

**Comment by alelazcano** [comment]
Ok, Looks like I fell asleep on the keyboard last night and made a mess. Everything is fixed now :) finally the tests are all ok and the PR for New Topic Prestashop is ready.

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Pull request merged**